applications/plugins/CMakeLists.txt

```diff
@@ -47,6 +47,8 @@ sofa_add_plugin(RigidScale RigidScale)
 sofa_add_plugin(LeapMotion LeapMotion)
 sofa_add_plugin(Geomagic Geomagic)
 sofa_add_plugin(SofaAssimp SofaAssimp) # ColladaSceneLoader Depends on Flexible and image
+sofa_add_plugin(LMConstraint LMConstraint)
```

Comment:
LMConstraint needs to be **before** BulletCollisionDetection if you need it to be able to find_package(LMConstraint)
Otherwise it will be false forever